src/foundry/common/constants.d.mts

```diff
@@ -857,7 +880,7 @@ export type SOFTWARE_UPDATE_CHANNELS = ValueOf<typeof SOFTWARE_UPDATE_CHANNELS>;
 /**
  * The default sorting density for manually ordering child objects within a parent
  */
-export declare const SORT_INTEGER_DENSITY: number;
+export declare const SORT_INTEGER_DENSITY: 100000;
```

Comment:
I'd probably leave this as `number`. Anyone who depends on this on being _precisely_ 4 is not being forward compatible and Foundry's allowed to change this on a whim.